docs/api/flink/Function.md

```diff
@@ -928,3 +928,33 @@ SELECT ST_ZMin(ST_GeomFromText('LINESTRING(1 3 4, 5 6 7)'))
 
 Output: `4.0`
 
+## ST_GeometricMedian
```

Comment:
Please place this to the correct place because functions are sorted alphabetically.